pages.de/common/grep.md

```diff
@@ -0,0 +1,36 @@
+# grep
+
+> Findet Ausdrücke in einem Eingabetext.
+> Unterstützt einfache Muster und reguläre Ausdrücke.
+
+- Suche nach einem Ausdruck in einer Datei:
+
+`grep {{ausdruck}} {{pfad/zu/datei}}`
+
+- Suche nach einem exakten Ausdruck:
+
+`grep -F {{exakter_ausdruck}} {{pfad/zu/datei}}`
+
+- Suche nach Ausdrücken [R]ekursiv im aktuellen Verzeichnis, zeige zugehörige Zeilen[n]ummern und [I]gnoriere Binärdateien:
+
+`grep -RIn {{ausdruck}} .`
+
+- Benutze erweiterte reguläre Ausdrücke (unterstützt `?`, `+`, `{}`, `()` und `|`) ohne Beachtung der Groß-, Kleinschreibung:
```

Comment:
I don't know German, but this `Beachtung der Groß-,` looks strange.  Is it definitely correct?

Reply:
That's correct

Reply:
Interesting, okay, thank you!